### PR TITLE
Rename private to local

### LINKED ISSUE #91: Consider avoiding the use of the word "private"
I know that local network address space is often referred to as "private" addresses (even in RFC 1918 itself), but this might be misleading.  These networks are not private, they just use a locally defined address space.  This does not confer any sort of privacy.

Also, this specification provide privacy for network access.

I suggest maybe "local" would be more appropriate, in line with the use in RFC 4291 in "link-local" and "site-local".  Though this is probably not entirely correct in that these address spaces are not always local, their address layout is locally defined.  "Local" is also an antonym for "global", which is what these addresses are not.

### PR DESCRIPTION
**Pull request by johnathan79717**
Fixes #91

### CODE REVIEW

**Comment by yoavweiss** [comment]
Hey @johnathan79717 - can you join the WICG to appease the IPR bots? :)

**Comment by johnathan79717** [comment]
Ah right.  I requested to join WICG just now.

I have kept the header names the same until we decide what to do with it.  I'll probably need to modify the explainers if we're gonna go through with this.  Did I miss anything else?

**Comment by annevk** [comment]
Please file explicit follow-up issues for things that are not renamed since you plan on closing the initial issue once this lands.

**Comment by johnathan79717** [comment]
> Please file explicit follow-up issues for things that are not renamed since you plan on closing the initial issue once this lands.

Good point.  I removed the keyword in the description so it should not close the initial issue now.

**Comment by johnathan79717** [comment]
Reopening this PR since renaming is back on the table.

**Comment by letitz** [comment]
Let me know when this is ready for review.

**Comment by johnathan79717** [comment]
> Let me know when this is ready for review.

It is ready for review. 😄

**Comment by letitz** [comment]
Perfect! I shall start reviewing then :)

**Review by letitz — CHANGES_REQUESTED** [review]
Looks good! I think we also want to update the base `explainer.md` file, right?

File: README.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
 
-# Private Network Access
+# Local Network Access
```
[the remark this answers is not in the document]
Comment (johnathan79717) [reply README.md:3]: Done

File: HOWTO.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ a pair of command-line flags:
 1. `--enable-features=PrivateNetworkAccessSendPreflights`
```
[the remark this answers is not in the document]
Comment (johnathan79717) [reply HOWTO.md:27]: Done

File: README.md [hunk 3]
```diff
@@ -1,6 +1,9 @@
 
 # Local Network Access
 
+Note: This specification has been renamed from *Private Network Access* to
+*Local Network Access*.
```
Comment (letitz) [inline README.md:5]:
```suggestion
*Local Network Access*. It was also initially known as *CORS-RFC1918*.
```

**Review by letitz — CHANGES_REQUESTED** [review]
Thanks very much! One last suggestion and we can merge.

File: README.md (hunk 3 above)
Comment (johnathan79717) [reply README.md]: Done

**Pull request merged**